src/Google/Ads/ServiceClientFactoryTrait.php

```diff
@@ -13,158 +13,25 @@
 
 use Google\Ads\GoogleAds\Constants;
 use Google\Ads\GoogleAds\Lib\ConfigurationTrait;
-use Google\Ads\GoogleAds\V9\Services\AccessibleBiddingStrategyServiceClient;
-use Google\Ads\GoogleAds\V9\Services\AccountBudgetProposalServiceClient;
 use Google\Ads\GoogleAds\V9\Services\AccountBudgetServiceClient;
 use Google\Ads\GoogleAds\V9\Services\AccountLinkServiceClient;
```

Comment:
The list was manually created by checking all the Services in the original file and comparing it with any of the ones we used within our code. Now that we switched to automatically detecting which ones we use, this would no longer be needed, which makes the script easier to maintain when switching library versions.